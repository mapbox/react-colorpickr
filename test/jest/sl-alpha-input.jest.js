import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as testCases from '../test-cases/sl-alpha-input-test-cases';

describe('SLAlphaInput', () => {
  let testCase;
  let wrapper;

  describe(testCases.basic.description, () => {
    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = shallow(React.createElement(testCase.component, testCase.props));
    });

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
