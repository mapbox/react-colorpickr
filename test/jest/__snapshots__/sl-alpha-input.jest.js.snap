// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SLAlphaInput basic renders 1`] = `
<NumberInput
  label="S"
  max={100}
  min={0}
  onChange={[Function]}
  theme={Object {}}
  value={92}
/>
`;
